api/app_analytics/tasks.py

```diff
@@ -62,14 +62,19 @@ def clean_up_old_analytics_data():
 
 
 @register_task_handler()
-def track_feature_evaluation(environment_id, feature_evaluations):
+def track_feature_evaluation(
+    environment_id: int,
+    feature_evaluations: dict[str, int],
+    identifier: Optional[str] = None,
```

Comment:
I _think_ the more recent approach for defining optional types is `identifier: str | None = None` ?

Reply:
Yeah you're right. This will take me so long to internalize. Feel free to keep correcting me on it.